platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/AddressBookInitializer.java

```diff
@@ -265,6 +269,22 @@ private void recordAddressBooks(@NonNull final AddressBook usedAddressBook) {
         }
     }
 
+    /**
+     * Deletes the oldest address book files if there are more than the maximum number of address book files.
+     */
+    private synchronized void cleanAddressBookDirectory() {
+        try {
+            List<Path> files = Files.list(pathToAddressBookDirectory).sorted().toList();
```

Comment:
Can be `final`. Also, stream returned by `Files.list` should be opened with try-with-resources pattern

Reply:
added final.